db/conn.go

```diff
@@ -16,7 +16,8 @@ package db
 
 import (
 	"encoding/json"
-	"fmt"
+	"github.com/rs/zerolog/log"
+	ulog "github.com/tigrisdata/fdb-exporter/util/log"
```

Comment:
This is very minor so feel free to ignore. I would wrap all logging in the new `util/log` package so that you don't have to import two logging things here.

Reply:
Something like `ulog.Error().Str` but this might be unnessecary

Reply:
I will keep it like this to maintain consistency with other repos, but very good point.